resources/lang/fr/flights.php

```diff
@@ -17,18 +17,18 @@
     'arr'              => 'Arr',
     'level'            => 'Niveau',
     'alternateairport' => 'Aéroport Alternatif',
-    'inbound'          => 'Vols à l\'arrivée',
-    'outbound'         => 'Vols au départ',
-    'none'             => 'Auncun vols trouvé.',
+    'inbound'          => 'Vols entrants',
+    'outbound'         => 'Vols sortants',
+    'none'             => 'Ce vol n\'existe pas.',
     'departuretime'    => 'Heure de Départ',
     'arrivaltime'      => 'Heure d\'Arrivée',
     'type'             => [
         'pass_scheduled'    => 'Passager (Prévu)',
-        'pass_addtl'        => 'Passager (Additionnel)',
+        'pass_addtl'        => 'Passager (Priv")',
```

Comment:
I'm not sure what's the meaning of "Passenger (Additional)" but in French the literal translation of "Additional" is "Additionnel" while "Privé" (there is a type the e is missing) means private

Reply:
My interpretation of additional is that this is a non scheduled/company flight, so I choose "privé" for "private flight" ??
And you're right, there i a tipo 'é missing"

Reply:
Well I don't really know the meaning in english so i'm not sure about how to translate it

Reply:
Pareil, pour moi un vol additionnel est un vol non régulier, et comme il existe déjà les vols charter ici, j'ai interprèté en "vol privé"

Reply:
C'est good pour moi, j'ai aucune idée de ce que ca peut vouloir dire je l'ai jamais vu dans le site...